### DISCUSSION TITLE
error install 3.4 on Monterey

### DISCUSSION THREAD

**Discussion by ArtemNic** [phalcon/cphalcon#15791] [Q&A] [answered]
Hello everyone! can't install phalcon 3.4 on macos Monterey. Need help. I am copying https://github.com/phalcon/cphalcon/tree/3.4.x. and trying to install using ./install
---------------------------
Result:
fatal error: too many errors emitted, stopping now [-ferror-limit=]
1 warning and 20 errors generated.
---------------------------
Php version 7.4
---------------------------
![image](https://user-images.githubusercontent.com/20792570/142147788-267a0bd3-d216-4230-a1b6-b99387c2ffe3.png)

**Comment by Jeckerson** [comment, marked as answer]
Phalcon v3.4 isn't supported in PHP 7.4, only `<= 7.3`

**Reply by ArtemNic** [reply]
Thx! It's work! PHP 7.2
